sdk-extensions/incubator/build.gradle.kts

```
@@ -22,12 +27,63 @@ dependencies {
   implementation(project(":sdk-extensions:autoconfigure-spi"))
   implementation("org.snakeyaml:snakeyaml-engine")
 
-  // io.opentelemetry.sdk.extension.trace.zpages
+  // io.opentelemetry.sdk.extension.incubator.trace.zpages
   implementation(project(":semconv"))
   compileOnly("com.sun.net.httpserver:http")
 
+  // io.opentelemetry.sdk.extension.incubator.fileconfig
+  implementation("com.fasterxml.jackson.core:jackson-databind")
+  implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
+
   testImplementation(project(":sdk:testing"))
   testImplementation(project(":sdk-extensions:autoconfigure"))
 
   testImplementation("com.google.guava:guava-testlib")
 }
+
+val configurationRef = "d018613a1bbac1ca5bbd6edbc7e6effeab3a71d0"
+val configurationRepoZip = "https://github.com/open-telemetry/opentelemetry-configuration/archive/$configurationRef.zip"
+
+val downloadConfigurationSchema by tasks.registering(Download::class) {
+  src(configurationRepoZip)
+  dest("$buildDir/configuration/opentelemetry-configuration.zip")
+  overwrite(false)
+}
+
+val downloadAndUnzipConfigurationSchema by tasks.registering(Copy::class) {
+  dependsOn("downloadConfigurationSchema")
```

Nit: wouldn't referring to the task itself work too? (and it'd be more type-safe too)
```suggestion
  dependsOn(downloadConfigurationSchema)
```